src/utils/helper.ts

```diff
@@ -333,7 +333,7 @@ export function parseResourceId(mpath: string): Map<string, string> {
     return ret;
 }
 
-export function findNodeInCodeModel(cliM4Path: any, codeModel: CodeModel, flattenMode: boolean = false) {
+export function findNodeInCodeModel(cliM4Path: any, codeModel: CodeModel, flattenMode: boolean = false, nodeTobeFound: any = null, noMatch: boolean = false) {
```

Comment:
>nodeTobeFound: any = null, noMatch: boolean = false [](start = 104, length = 51)

what's the meaning of those two parameters?

Reply:
nodeTobeFound is the node I want to found it's from python code model. noMatch is means the cli path and the cliNode is from the same code model and there's no need to go through the match logic.